WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -265,6 +266,11 @@
     <hr>
     <%= f.input :remarks, disabled: is_actually_confirmed %>
 
+    <% if @competition.new_record? && @competition.cloned_from && !@competition.cloned_from.competition_tabs.empty? %>
```

Comment:
To be more concrete, I was thinking something like having `clone.clone_tabs = competition_tabs.present?` in `clone_tabs()`.

I assumed the page is only rendered once, so if the user unchecks the box that `false` value will never render again, but on second thought, the page will be rerendered if there's a validation error, and that breaks my fragile plan. It's kinda hacky anyway.

How about this:

`@competition.cloned_from && !@competition.cloned_from.competition_tabs.empty?` should be the same as `@competition.cloned_from&.competition_tabs&.present?`.

I also think `cloned_from` is defined if and only if `@competition.new_record?` is true, so it could just be `if @competition.cloned_from&.competition_tabs&.present?`

And maybe it should be called `being_cloned_from` to show it's an temporary attribute for form purposes only.